### PR TITLE
Create Documentation for Lando Setup Guide With Pantheon

### PR DESCRIPTION
**Pull request by digisavvy**
## Effect
PR includes the following changes:
- Initial document submitted for review.

## Remaining Work
- [ ] Technical review
- [ ] Copy Review

## Post Launch
To be completed by the docs team upon merge: 
- [ ] Include/exclude pages ^ respectively within docs search service provider (if applicable)
- [ ] Update Status Report
- [ ] Archive from **Done** in Waffle

### CODE REVIEW

File: source/_docs/guides/lando.md [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+
+
+Installing Lando is straight forward. Be sure to check its [system requirements](https://docs.devwithlando.io/installation/system-requirements.html){.external} to ensure you can proceed.
+
+## Downloading and Install Lando
+
+- The latest release of Lando for your OS can be found [here](https://github.com/lando/lando/releases){.external}. Download the appropriate release for your Operating System.
+- Click to open the installer package. The installer has a few pre-flight checks it runs before getting started.
+
+![Image of the Lando installer](/source/docs/assets/images/guides/lando/IMAGE.png "Lando Installer")
```
Comment (alexfornuto) [inline source/_docs/guides/lando.md]: @digisavvy I couldn't access the Google URLs. Add your images to `/source/docs/assets/images/guides/lando/`

File: source/_docs/guides/lando.md [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+
+1. The `lando wp core download` command downloads the current stable release of WordPress and unpacks it in your current working directory.
+
+1. Your WP configuration file still needs to be created. You can actually do this from the command line. However, we’re going to go to do this the ol’ fashioned way through the browser.
+
+1. In your browser, enter in the URL given to you above: _http://yourappname.lndo.site_. You’ll notice WordPress’s _Famous Five Minute install_ screen. 
+
+1. Click next after choosing your preferred language and click Let’s Go on the following screen.
+
+1. In Lando, for a WordPress installation without Pantheon, the Database, Username, and Password are all `WordPress` and Database Host is `database` (the database host is case-sensitive). Enter your credentials and complete the installation process! Congrats! You now have Lando installed with WordPress!
```
Comment (alexfornuto) [inline source/_docs/guides/lando.md]: This is where my testing failed. I used the values provided but was unable to connect to the database.

**Comment by alexfornuto** [comment]
@digisavvy any update? Should we close this?

**Comment by digisavvy** [comment]
> @digisavvy any update? Should we close this?

I'm going to get this updated asap. Thank you!

**Comment by digisavvy** [comment]
Hey @alexfornuto - I pushed some updates here for review when you get a chance. Sorry for the lag.

**Comment by alexfornuto** [comment]
Thanks @digisavvy! I've provided a PR for you [here](https://github.com/digisavvy/documentation/pull/1) - I also realized that my comments from another PR were never submitted so you can see them, so I'll re-note them here.

File: source/_docs/guides/lando.md [hunk 3]
```diff
@@ -0,0 +1,165 @@
...
+
+1. The `lando wp core download` command downloads the current stable release of WordPress and unpacks it in your current working directory.
+
+1. Your WP configuration file still needs to be created. You can actually do this from the command line. However, we’re going to go to do this the ol’ fashioned way through the browser.
+
+1. In your browser, enter in the URL given to you above: _http://yourappname.lndo.site_. You’ll notice WordPress’s _Famous Five Minute install_ screen.
+
+1. Click next after choosing your preferred language and click Let’s Go on the following screen.
+
+1. In Lando, for a WordPress installation without Pantheon, the Database, Username, and Password are all `wordpress` and Database Host is `database` (the all values are case-sensitive). Enter your credentials and complete the installation process! Congrats! You now have Lando installed with WordPress!
```
Comment (alexfornuto) [inline source/_docs/guides/lando.md]: Using these credentials, I'm unable to establish a connection to the database.

image

I haven't found anything in Lando's docs to explain it. @dustinleblanc do you have any input?

Comment (alexfornuto) [reply source/_docs/guides/lando.md]: Dustin and I figured it out, error on my par.

**Comment by alexfornuto** [comment]
Having finished my review and successfully tested all steps, we can proceed without the need to merge my PR.

**Comment by digisavvy** [comment]
Heya, Alex!
That's great news. I reviewed the preview Katie sent me. It looks good. Are you
waiting on me for anything else?







On Fri, Sep 20, 2019 4:36 PM, Alex Fornuto notifications@github.com  wrote:
Having finished my review and successfully tested all steps, we can proceed
without the need to merge my PR.

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub, or mute the thread.



Alex Vasquez (he/him)| Principal
DigiSavvy, Inc.alex@digisavvy.com|digisavvy.com714.989.6654| @alexjvasquez

**Comment by alexfornuto** [comment]
Nope, nothing needed, the guide is [live](https://pantheon.io/docs/guides/lando)! It will be added to the landing pages shortly.

This PR _should_ show as merged now, but I assume that there are commits here not included in the merged branch. I'll see if I can align them.

**Pull request closed without merging**